Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.h

```diff
@@ -37,8 +37,6 @@ namespace AzNetworking
 
 namespace Multiplayer
 {
-    AZ_CVAR_EXTERNED(AZ::CVarFixedString, sv_defaultPlayerSpawnAsset);
```

Comment:
Why not extern this where you need it, see my other review comment.

Reply:
I removed extern because externing won't pick up the value set in setreg.

Reply:
😕 😖 indeed. i'll maybe poke into how that setreg works later if no one picked up LYN-8014. to make it even more interesting, if you type the cvar out in the editor console, it'll still show me correct value that matches what I defined in the setreg, but if I put a code break when the externed value is used, it'll be the old default value.